MapAnalyzer/MapData.py

```diff
@@ -198,6 +198,15 @@ def find_lowest_cost_points(self, from_pos: Point2, radius: float, grid: np.ndar
 
         """
         return self.pather.find_lowest_cost_points(from_pos=from_pos, radius=radius, grid=grid)
+    
+    def lowest_cost_points_array(self, from_pos: Point2, radius: float, grid: np.ndarray) -> ndarray:
```

Comment:
Please add to the docstring  why this functions is needed,  an example use case
and also time comparison to older method.
if i recall correctly it improves performance by ~ 25%   but this needs to be logged in the code for future reference